src/components/ButtonPanel.js

```diff
@@ -3,34 +3,34 @@ import Button from './Button';
 
 function ButtonPanel() {
   return (
-    <div>
-      <div>
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
+    <div className="buttonPanel">
+      <div className="buttonGroup">
+        <Button name="AC" color="#e0e0e0" />
+        <Button name="+/-" color="#e0e0e0" />
+        <Button name="%" color="#e0e0e0" />
         <Button name="÷" />
       </div>
-      <div>
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
+      <div className="buttonGroup">
+        <Button name="7" color="#e0e0e0" />
+        <Button name="8" color="#e0e0e0" />
+        <Button name="9" color="#e0e0e0" />
         <Button name="X" />
       </div>
-      <div>
-        <Button name="4" />
-        <Button name="5" />
-        <Button name="6" />
+      <div className="buttonGroup">
+        <Button name="4" color="#e0e0e0" />
+        <Button name="5" color="#e0e0e0" />
+        <Button name="6" color="#e0e0e0" />
         <Button name="-" />
       </div>
-      <div>
-        <Button name="1" />
-        <Button name="2" />
-        <Button name="3" />
+      <div className="buttonGroup">
+        <Button name="1" color="#e0e0e0" />
+        <Button name="2" color="#e0e0e0" />
+        <Button name="3" color="#e0e0e0" />
         <Button name="+" />
       </div>
-      <div>
-        <Button name="0" />
-        <Button name="." />
+      <div className="buttonGroup">
+        <Button name="0" color="#e0e0e0" wide="true" />
```

Comment:
Wide prop should be a boolean, not a string. As stated here 
![image](https://user-images.githubusercontent.com/46115725/93952681-119c1580-fd0f-11ea-91f6-fd07ebe1d2e1.png)